packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -0,0 +1,315 @@
+        },
+        image () {
+            return this.card.image;
+        },
+        url () {
+            return this.card.url;
+        },
+        discountPercentage () {
+            return this.card.discountPercentage;
+        },
+        earnedStamps () {
+            return this.card.earnedStamps;
+        },
+        expiryDate () {
+            return this.card.expiryDate;
+        },
+        expiryLine () {
+            return this.card.expiryLine;
+        },
+        isReadyToClaim () {
+            return !!this.card.isReadyToClaim;
+        },
+        totalRequiredStamps () {
+            // TBC by product - currently this.card.totalRequiredStamps is provided but we have no way of catering for
+            // any amount other than 5 here
+            return 5;
+        },
+        stamps () {
+            const stamps = [];
+
+            for (let i = 0; i < this.totalRequiredStamps; i++) {
+                const full = (i < this.earnedStamps);
+                stamps.push(full ? {
+                    stampImage: 'FullStamp',
+                    classSuffix: 'Full'
+                } : {
+                    stampImage: 'EmptyStamp',
+                    classSuffix: 'Empty'
+                });
+            }
+
+            return stamps;
+        },
+        statusText () {
+            return this.card.subtitle;
+        },
+        subStatusText () {
+            return this.description;
+        },
+
+        testIdForSection () {
+            return (section, index) => (this.testId
+                ? `${this.testId}--${section}${index === undefined ? '' : `--${index}`}`
+                : false);
+        }
+    },
+
+    mounted () {
+        this.onViewContentCard();
+    },
+
+    inject: [
+        'emitCardView',
+        'emitCardClick'
+    ],
+
+    methods: {
+        onViewContentCard () {
+            this.emitCardView(this.card);
+        },
+
+        onClickContentCard () {
+            this.emitCardClick(this.card);
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+
+$stampCard-subStatus-colour: #017a39; /* $color-green in PIE - not in fozzie-colour-palette yet */
+$stampCard-expiryInfo-colour: $grey--dark;
+
+$stampCard-responsive-mobileViewBreakpoint: '<narrowMid';
+
+.c-stampCard1 {
+    text-decoration: initial;
+    width: 392px;
+    display: flex;
+    flex-direction: column;
+    padding: spacing(x2);
+    border-radius: $border-radius;
+    box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.03),
+    0 3px 1px -2px rgba(0, 0, 0, 0.07),
+    0 1px 5px 0 rgba(0, 0, 0, 0.06);
+
+    &,
+    &:hover,
+    &:visited,
+    &:focus {
+        color: $color-text;
+    }
+
+    @include media($stampCard-responsive-mobileViewBreakpoint) {
+        width: auto;
+        box-shadow: 0 0 6px 0 rgba(0, 0, 0, 0.12);
+    }
+}
+
+.c-stampCard1-headerDetails {
+    margin-bottom: spacing();
+}
+
+.c-stampCard1-icon {
+    float: left;
+    margin-right: spacing(x2);
+    width: spacing(x7);
```

Comment:
For width and height it is better to use px or variables than spacing, which is used for paddings and margins